### DISCUSSION TITLE
Is Copilot's client open source?

### DISCUSSION THREAD

**Discussion by meddion** [microsoft/vscode-discussions#2096] [Extension Development QnA] [answered]
I'd like to change some things in the UI of Copilot in VS Code, as well as look at some prompts if it is available. Where can I find the source code if any?

**Comment by alefragnani** [comment, marked as answer]
Hi @meddion ,

No, GitHub Copilot is not open source.

if you have suggestions to improve the extension, I would suggest you to follow the channels described here https://code.visualstudio.com/docs/copilot/faq#_how-can-i-provide-feedback-on-copilot.

Hope this helps
